packages/client/src/validate.js

```diff
@@ -0,0 +1,14 @@
+import isEmpty from 'lodash/isEmpty';
+
+export default {
+  logIn(accountsClient, { user, password }) {
```

Comment:
Having this kind of function here means we are sticking with the password method as a the main authentication method. If this is used in the react package, I am not sure really how to break it down to phases